traffic_ops/app/lib/Schema/Result/DeliveryServiceInfoForCdnList.pm

```diff
@@ -0,0 +1,101 @@
+
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+#
+
+package Schema::Result::DeliveryServiceInfoForCdnList;
+
+# this view returns the regexp set for a delivery services, ordered by type, set_number.
+# to use, do
+#
+# $rs = $self->db->resultset('DeliveryServiceInfoForCdnList')->search({}, { bind => [ $id ]});
+#
+# where $id is the CDN id.
+
+use strict;
+use warnings;
+use utf8;
+use base 'DBIx::Class::Core';
+
+__PACKAGE__->table_class('DBIx::Class::ResultSource::View');
+
+__PACKAGE__->table("DeliveryServiceInfoForCdnList:");
+
+__PACKAGE__->result_source_instance->is_virtual(1);
+
+__PACKAGE__->result_source_instance->view_definition( "
+SELECT DISTINCT
```

Comment:
I'm a bit confused why there is a DeliveryServiceInfoForCdnList.pm and a DeliveryServiceInfoForDomainList.pm. the 2nd one already exists. based on their names, they sound the same to me....